typings/atom/index.d.ts

```diff
@@ -1,12 +1,19 @@
 import { Point, Notification, NotificationOptions, TextEditor } from "atom"
 
```

Comment:
The lint error might go away by using `export {}` or exporting the interfaces

Reply:
But... the lint is plain wrong, isn't it?

Reply:
After exporting, eslint complains about name shadowing, which is just stupid. It doesn't seem to understand what module augmentations are.

Reply:
There are many, many other eslint failures here. Fixing those is out of scope.